### ISSUE TITLE
Guidance required

### ISSUE THREAD

**Issue by koltegirish** [marty-Wallace/Pothole#1]
Hello marty,
                 Can you provide in detail the installation steps for each python module and which version of python and opencv is used. 
Highly appreciated  for helping in completing the pothole project.

**Comment by marty-Wallace** [comment]
Hi @koltegirish, 
This is currently a project I am working on for college. Currently the only dependacy is Python Image Library which can be installed by running the command `pip install pillow`. The project is written in python 3.6 (it may run under python2 but I will make no guarentees).

 One of the requirements for the class project was to not rely on any libaries (PIL was fine though). Once the school project is complete I would like to refactor the project to use numpy and/or opencv. It would likely see an in speed if the processing functions can be c libraries.

This project is still under developement and will most likely be refactored pretty heavily over the next week or so. I wouldn't count on any of this staying the same. Once I am happier with the results of the project and it feels a little more stable then I will write up a set of formal instructions on how to use it as well as a cleaner CLI interface. 

Let me know if you have any other questions/suggestions and feel free to open up another issue if you'd like.
